server/protocols/protocols-imap4/src/main/java/org/apache/james/imapserver/netty/IMAPServer.java

```diff
@@ -99,7 +109,27 @@ public void doConfigure(HierarchicalConfiguration configuration) throws Configur
         if (timeout < 0) {
             timeout = 0;
         }
-        
+
+        processor.configure(getImapConfiguration(configuration));
+    }
+
+    @VisibleForTesting static ImapConfiguration getImapConfiguration(HierarchicalConfiguration configuration) {
+        ImmutableSet<String> disabledCaps = ImmutableSet.copyOf(Splitter.on(CAPABILITY_SEPARATOR).split(configuration.getString("disabledCaps", "")));
+
+        return ImapConfiguration.builder()
+                .idleTimeInterval(configuration.getLong("idleTimeInterval", IdleProcessor.DEFAULT_HEARTBEAT_INTERVAL_IN_SECONDS))
```

Comment:
There is no way anymore to disable HeartBeat... (it was done before with a negative interval)

You should maybe add a configuration option **enableIdle**.